report/source/Main/Pilot_Program_Recap.md

```diff
@@ -1,186 +1,209 @@
 
-In fact, one key decision made in the development of the final Security Service Architecture Maturity rubric was that better architectures relied as much as possible on dedicated and isolated components distinguishable at the system level. This belief is baked into the rubric. Realizing that this is not possible for all security services, the rubric has accommodations for the security services we believed required some level of configuration or integration with the solution's custom software. We called these composite services, and the ones which could be completely isolated we called transparent services.
+In fact, one key decision made in the development of the final Security Service Architecture Maturity rubric was that better architectures rely as much as possible on dedicated and isolated components distinguishable at the system level. Realizing that this is not possible for all security services, the rubric has accommodations for the security services we believed required some level of configuration or integration with the solution's custom software. We called these composite services, an example of which is a Security Information and Event Management service, while those that we believe should be completely isolated we called transparent services, such as a firewall.
 
-The rubric also considers the complex nature of systems with components and sub-components. It does this by calculating scores at five layers, starting at the most detailed level of security service implementation per component or interface and resulting in a master architecture score. Then, the rubric has a unique category for *depth* which is intended to be top-down view over the whole architecture. In this manner, the architecture maturity score is a reflection from the bottom-up and a top-down perspectives.
+The rubric also considers the complex nature of systems with components and sub-components. It does this by calculating scores at five layers, starting at the most detailed level of security service implementation per component or interface and resulting in a master architecture score. Then, the rubric has a unique category for *depth*, a top-down view over the whole architecture. In this manner, the architecture maturity score is a reflection of both bottom-up and top-down perspectives.
```

Comment:
I'm not sure I understand what we are saying about `depth` here.

Reply:
I was struggling a bit with the takeaway here too. @wilsoaa?

Reply:
Unless thing changed or I never understood to begin with :), the architecture rubric is most bottom up. By that, I mean that the scores are built at the component level and then aggregated up multiple levels. However, there was also a top down category in the rubric. I thought it was called Depth. I went to find the Excel file from Brian in the Supporting folder - the nice one that does all the architecture calculations. I realized that I couldn't find it because it was too much detail to share. Anyway, if you look at that Excel, I think you will see what I was trying to say.

Reply:
got it. @JDziurlaj I'd appreciate any feedback you have. Also please link me to that file in Teams when you get a chance? Thanks!

Reply:
@wilsoaa This makes sense to me. I think the text can remain as-is. Thanks!